src/tests/multihost/ad/test_ad_misc.py

```diff
@@ -281,3 +281,45 @@ def test_0003_gssapi_ssh(self, multihost, adjoin, create_aduser_group):
             pytest.fail("Ssh login failed.")
             ssh_output = 'FAIL'
         assert 'ssh_result:0' in ssh_output, "GSSAPI ssh authentication failed"
+
+    @pytest.mark.tier1_3
+    @staticmethod
+    def test_0004_bz2110091(multihost, adjoin, create_aduser_group):
+        """
+        :title: SSSD starts offline after reboot
+        :id: 8fe03cef-891d-4cb9-be26-d747cb4d8fd8
+        :customerscenario: true
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=2110091
+                   https://bugzilla.redhat.com/show_bug.cgi?id=2116207
+        :steps:
+          1. Join sssd client to AD
+          2. Lookup AD user
+          3. Clear sssd cache and logs
+          4. Reboot client
+          5. Domain logs should have "Destroying the old c-ares channel"
+          6. Domain logs should have "[recreate_ares_channel]: Initializing new c-ares channel"
```

Comment:
test did not check for 2 times

Reply:
Log is checked only once and that too after reboot.